app/assets/javascripts/index/new_note.js

```diff
@@ -152,6 +152,16 @@ OSM.NewNote = function (map) {
       .on("input", updateControls)
       .focus();
 
+    var anonymousNotesCount = Number(localStorage.getItem("anonymousNotesCount"));
```

Comment:
```suggestion
    var anonymousNotesCount = Number(localStorage.getItem("anonymousNotesCount") ?? 0);
```
after #5421 is merged